firefox-ios/Client/ApplicationHelper.swift

```diff
@@ -46,4 +47,18 @@ struct DefaultApplicationHelper: ApplicationHelper {
             open(url)
         }
     }
+
+    func closeTab(_ url: URL) async {
+        // We need to get all the windows/tabs and then find the url that was requested to close
+        await UIApplication.shared.connectedScenes.forEach { scene in
+            let windowManager = AppContainer.shared.resolve() as WindowManager
+            windowManager.allWindowTabManagers().forEach { tabManager in
```

Comment:
Most of the heavy lifting here -- not sure if this is the best way, but I'll let a FxIOS dev chime in here!